### DISCUSSION TITLE
Unit in TFEL Glossary

### DISCUSSION THREAD

**Discussion by leopeuziat** [thelfer/tfel#270] [Q&A] [answered]
On the page http://tfel.sourceforge.net/glossary.html a unit is defined for each entry of the Glossary. Is is someting implemented in the code ? Is is possible to get it from python binding ?

**Comment by thelfer** [comment, marked as answer]
@leopeuziat This refers to recent developments in `TFEL`.

More precisely, units are attached to entries for a given unit system (with the SI unit system being the only supported right now). See those issues:

- https://github.com/thelfer/tfel/issues/255
- https://github.com/thelfer/tfel/issues/239
- https://github.com/thelfer/tfel/issues/237

The following code just works:

```python
import tfel.glossary
g = tfel.glossary.Glossary.getGlossary()
print(g.YoungModulus.units)
```
